driver-moab/main/prep_ocn_mod.F90

```diff
@@ -837,85 +837,85 @@ subroutine prep_ocn_init(infodata, atm_c2_ocn, atm_c2_ice, ice_c2_ocn, rof_c2_oc
 
 #endif
 
-          if (iamroot_CPLID) then
-             write(logunit,*) ' '
-             write(logunit,F00) 'Initializing mapper_Rr2o_ice'
-          end if
-          ! is this the same map as above ?
-          call seq_map_init_rcfile(mapper_Rr2o_ice, rof(1), ocn(1), &
-               'seq_maps.rc', 'rof2ocn_ice_rmapname:', 'rof2ocn_ice_rmaptype:',samegrid_ro, &
-               'mapper_Rr2o_ice  initialization', esmf_map_flag, no_match )
+         if (iamroot_CPLID) then
+            write(logunit,*) ' '
+            write(logunit,F00) 'Initializing mapper_Rr2o_ice'
+         end if
+         ! is this the same map as above ?
+         call seq_map_init_rcfile(mapper_Rr2o_ice, rof(1), ocn(1), &
+            'seq_maps.rc', 'rof2ocn_ice_rmapname:', 'rof2ocn_ice_rmaptype:',samegrid_ro, &
+            'mapper_Rr2o_ice  initialization', esmf_map_flag, no_match )
 ! us the same one for mapper_Rr2o_ice and mapper_Fr2o
 #ifdef HAVE_MOAB
```

Comment:
We should remove all HAVE_MOAB checks under `driver-moab` sources. Not for this PR - but it just occurred to me.